### PR TITLE
Data Exporter CSV Test

### PR DESCRIPTION
**Pull request by SASUKE38**
Test for DataExporterCSV header functionality.

### CODE REVIEW

File: test/org.jkiss.dbeaver.test.platform/src/org/jkiss/dbeaver/tools/transfer/DataExporterCSVTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import org.jkiss.dbeaver.model.data.DBDAttributeBinding;
```
Comment (E1izabeth) [inline test/org.jkiss.dbeaver.test.platform/src/org/jkiss/dbeaver/tools/transfer/DataExporterCSVTest.java:19]: Please add copyright as it's done in other files.

Comment (E1izabeth) [reply test/org.jkiss.dbeaver.test.platform/src/org/jkiss/dbeaver/tools/transfer/DataExporterCSVTest.java:19]: Also package declaration is missing

Comment (SASUKE38) [reply test/org.jkiss.dbeaver.test.platform/src/org/jkiss/dbeaver/tools/transfer/DataExporterCSVTest.java:19]: I have added these in the latest commit.

**Review by ShadelessFox — APPROVED** [review]

**Comment by serge-rider** [comment]
Thanks!

**Pull request merged**